Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```
@@ -32,3 +32,15 @@
     contents:
     - id: BoxSurvivalMedical
     - id: RubberStampPsychologist
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackCaptain
+  id: ClothingBackpackAdminAssistantFilled
+  name: administrative assistant's backpack
+  description: Its a backpack build for carrying paper.
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
+      - id: RubberStampAdminAssistant
```

```suggestion
      - id: RubberStampAdminAssistant

```
files should have a line at the end, it's good form supposedly